contentcuration/contentcuration/management/commands/exportchannel.py

```diff
@@ -263,6 +263,8 @@ def create_associated_thumbnail(ccnode, ccfilemodel):
         encoding = ccnode.thumbnail_encoding and load_json_string(ccnode.thumbnail_encoding).get('base64')
     except ValueError:
         logging.error("ERROR: node thumbnail is not in correct format ({}: {})".format(ccnode.id, ccnode.thumbnail_encoding))
+    except IOError:
+        logging.error("ERROR: cannot identify the thumbnail ({}: {})".format(ccnode.id, ccnode.thumbnail_encoding))
```

Comment:
Might be good to just return None here as it might cause problems if it continues. If it does return None, the thumbnail will default to the filemodel: https://github.com/learningequality/studio/blob/c1538dc50e96cfda3cfbc90c1ca9af42b1055351/contentcuration/contentcuration/management/commands/exportchannel.py#L296